tests/compiler_test.py

```diff
@@ -23,13 +23,18 @@ def loadUFO(filename):
 class CompilerTest(unittest.TestCase):
     _tempdir, _num_tempfiles = None, 0
 
-    def testTestFont(self):
+    def test_TestFont(self):
         # We have specific unit tests for CFF vs TrueType output, but we run
         # an integration test here to make sure things work end-to-end.
         # No need to test both formats for every single test case.
         self.expectTTX(compileTTF(loadUFO("TestFont.ufo")), "TestFont.ttx")
         self.expectTTX(compileOTF(loadUFO("TestFont.ufo")), "TestFont-CFF.ttx")
 
+    def test_regressions(self):
```

Comment:
or did you name it generically "regressions" because you plan on adding more test cases to the same test method?
I'd prefer to keep each unit test method separate, so we know from the test runner log what is failing